roboplot/imgproc/path_following.py

```diff
@@ -0,0 +1,28 @@
+import operator
+import roboplot.config as config
+import roboplot.imgproc.image_analysis as image_analysis
+
+
+def convert_to_global_coords(points, scan_direction, camera_position):
+
+    # Scale factors
+    x_scaling = config.X_PIXELS_TO_POINTS_SCALE  # (based on a 200x200 pixel photo of a 40mm square)
+    y_scaling = config.Y_PIXELS_TO_POINTS_SCALE  # (based on a 200x200 pixel photo of a 40mm square)
```

Comment:
Given that these are coming from the `config` file, I don't think you want the comments here do you?